corehq/apps/userreports/tasks.py

```diff
@@ -80,15 +80,21 @@ def rebuild_indicators(indicator_config_id):
         redis_key = _get_redis_key_for_config(config)
 
     adapter.rebuild_table()
-    relevant_ids = get_doc_ids_in_domain_by_type(
-        config.domain,
-        config.referenced_doc_type,
-        database=couchdb,
-    )
-    for docs in chunked(relevant_ids, 1000):
-        redis_client.sadd(redis_key, *docs)
-
-    _build_indicators(indicator_config_id, relevant_ids)
+    relevant_ids_chunk = []
+    for relevant_id in iterate_doc_ids_in_domain_by_type(
```

Comment:
i don't think that's exactly the way couch pagination works. you send it a startkey_docid (not an offset) so it picks up at the exact key where it left off. i'm not sure what the behavior is if the startkey_docid gets deleted as you are making a request for it. maybe @dannyroberts would know, but seems like an edge case (though plausible)